www/client/components/SubSection/index.js

```diff
@@ -1,65 +1,100 @@
-          className='docs-sub-section__body copy-spacing'
-          dangerouslySetInnerHTML={{ __html: subSection.subSectionBodyContent }}
+          className="docs-sub-section__body copy-spacing"
+          dangerouslySetInnerHTML={{ __html: subSectionBodyContent }}
         />
         {/* eslint-enable react/no-danger */}
-        {
-          (subSection.subSectionImage || subSection.subSectionImageDescription)
-          &&
+        {(subSectionImage || subSectionImageDescription) && (
           <div
             className={
               'flex-row' +
-              `${
-                subSection.subSectionImagePosition
-                &&
-                ` flex-row--${
-                    subSection.subSectionImageDescription
-                    ?
-                    subSection.subSectionImagePosition
-                    : 'center'
-                }` || ''
-              }`
+              `${(subSectionImagePosition &&
+                ` flex-row--${subSectionImageDescription ? subSectionImagePosition : 'center'}`) ||
+                ''}`
             }
           >
-            {
-              subSection.subSectionImage
-              &&
-              <div className={`docs-sub-section__image-section medium-${subSection.subSectionImageSize} columns`}>
-                <img src={subSection.subSectionImage} alt={subSection.subSectionImageDescription} />
+            {subSectionImage && (
+              <div
+                className={`docs-sub-section__image-section medium-${subSectionImageSize} columns`}
+              >
+                <img src={subSectionImage} alt={subSectionImageDescription} />
               </div>
-            }
-            {
-              subSection.subSectionImageDescription
-              &&
+            )}
+            {subSectionImageDescription && (
               <div
-                className={`medium-${12 - subSection.subSectionImageSize} columns`}
+                className={`medium-${12 - subSectionImageSize} columns`}
                 /* eslint-disable react/no-danger */
-                dangerouslySetInnerHTML={{ __html: subSection.subSectionImageDescription }}
+                dangerouslySetInnerHTML={{ __html: subSectionImageDescription }}
                 /* eslint-enable react/no-danger */
               />
-            }
+            )}
+          </div>
+        )}
+        {subSectionImageTabs && (
+          <div>
+            <Tabs>
+              <TabList>
+                {subSectionImageTabs.map((tab, idx) => (
+                  <Tab
+                    key={`${tab.sectionImageTabLabel}-${idx}`}
+                    heading={tab.sectionImageTabLabel}
+                  />
+                ))}
+              </TabList>
+              <TabContent>
+                {subSectionImageTabs.map((tab, idx) => (
+                  <TabPane key={`${tab.id}-${idx}`}>
+                    <img
+                    src={tab.subSectionImageTabArray.url} alt={tab.sectionImageTabLabel}
```

Comment:
interesting tabbing here - maybe tab both props and add a line return

Reply:
I have another PR. I'll fix it there.